### ISSUE TITLE
Auth code malformed

### ISSUE THREAD

**Issue by CKL1976** [AdyRock/com.mercedes_me#1]
Hi, thanks for the cool ME app :-)

I am trying to get it working but keep getting an error stating Authentication code is malformed.

Any suggestions?

**Comment by AdyRock** [comment]
Have you seen the setup guide on the forum and been through the setup of the Mercedes side of the app?
Make sure you have added all of the APIs that are shown as that can cause that error.

**Comment by CKL1976** [comment]
Yes, did go through all the settings and have all 5 Api´s enabled..

Also tried to reinstall both the Homey app, but also the devleoper Mercedes part. But still same result

Does the project name need to be a specific name or could it be anything ?

**Comment by AdyRock** [comment]
After to save the ID, Secret and VIN can you restart the app to see if that fixes it?

**Comment by CKL1976** [comment]
> After to save the ID, Secret and VIN can you restart the app to see if that fixes it?

I will check :-)

**Comment by CKL1976** [comment]
It didnt help :-(..... Still the same.

**Comment by AdyRock** [comment]
Hmm, this is the first problem as it's a new app so it might take some investigations to get to the bottom of this.

I have uninstalled and reinstalled the app on my Homey from the Athom store, so it is a fresh start. I entered my details again and then added a car and it all went without issues. So it might be difficult to find this one.

Could you post a screenshot of the Mercedes Me setup screen (without showing the Client ID and Secret).

Edit: I have added a screenshot of my Mercedes me page to the instruction in the forum.

**Comment by AdyRock** [comment]
When you enter the Client ID and Secret are you typing it in or copy and past?

**Comment by CKL1976** [comment]
I use copy/paste and have tried from both phone and chrome browser on Mac

**Comment by AdyRock** [comment]
When you try to add the car, have you seen the Mercedes window asking you to allow access or does the error appear before that?

**Comment by CKL1976** [comment]
This is how it looks :-)

**Comment by CKL1976** [comment]
It works....

I added the Connected Vehicle Sandbox, so these are the enabled API’s



And then the Mercedes Authentication showed up 👍🏻

**Comment by AdyRock** [comment]
OK, strange but good. 
I added that at a later stage to experiment. When I noticed it was still there I removed it again then went through the process of uninstall install etc and it was still OK. 
But at least I know what to suggest to anyone else that has a problem so thanks for letting me know.

**Comment by CKL1976** [comment]
Happy to help out :-)

I will let you know if anything strange happens, but it looks pretty cool so far 😊
